### PR TITLE
NO-ISSUE: Forbid concurrent execution of all cronjobs

### PR DESCRIPTION
**Pull request by danmanor**
Forbid concurrent execution of all cronjobs

### CODE REVIEW

File: openshift/template.yaml [hunk 1]
```diff
@@ -221,6 +223,7 @@ objects:
         backoffLimit: 4
         template:
           spec:
+            concurrencyPolicy: ${JOBS_AUTO_REPORT_CONCURRENCY_POLICY}
```
Comment (adriengentil) [inline openshift/template.yaml:226]: should be above in the `spec` no?

Comment (danmanor) [reply openshift/template.yaml]: Yes thanks

**Review by adriengentil — APPROVED** [review]

**Pull request merged**